resources/assets/sass/components/_title.scss

```diff
@@ -0,0 +1,30 @@
+// Title
+//
+// Title component
+//
+// Markup:
+//<h2 class="c-title {$modifiers}">Tripikad räägivad</h2>
+//
+// .m-yellow - Yellow
+// .m-green - Green
+// .m-red - Red
+// .m-blue - Blue
+// .m-purple - Purple
+// .m-orange - Orange
+//
+// Styleguide 9
+
+.c-title {
+
+	@include color-modifiers;
+
+	font-size: $font-size-small;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered font-family, font-size, font-weight, text-transform